deployments/demos/zenoh-bridge/README.md

```diff
@@ -35,6 +41,39 @@ cd local
 
 Then access the visualizer at `http://localhost:6081`.
 
+
+
+## Distributed Deployment (Multi-Machine)
+
+To deploy on separate machines (e.g., one Cloud, one Edge):
+
+1.  **Cloud Machine**:
+    Run `cloud.sh` to start services. It will automatically detect and display available IP addresses:
+    ```bash
+    ./cloud.sh
+    # [Info] Cloud services started.
+    #        To connect from Edge, set CLOUD_IP to one of the following:
+    #        [Public/Routable IPs]
+    #        - 192.168.1.100
+    ```
+
+2.  **Edge Machine**:
+    Use the displayed IP to connect:
+    ```bash
+    export CLOUD_IP=192.168.1.100
```

Comment:
It would be great to also put the CLOUD_IP information in the official document
https://autowarefoundation.github.io/openadkit/deployments/zenoh-bridge/